projects/step-frontend/src/lib/angularjs/js/ntables.js

```diff
@@ -378,7 +380,8 @@ angular
                   if (scope.serverSideParameters) {
                     data.params = scope.serverSideParameters;
                   }
-                  return JSON.stringify(tableLegacyUtils.transformRequestLegacy2New(data));
+                  lastServerSideRequest = tableLegacyUtils.transformRequestLegacy2New(data);
```

Comment:
@dvladir Do we still have legacy tables?

Reply:
@jeromecomte Yes. 
For example assign entities modal, that belongs to angularjs part uses legacy table